prboom2/src/m_file.c

```diff
@@ -545,3 +550,46 @@ char *M_getenv(const char *name) {
   return getenv(name);
 #endif
 }
+
+static int nftwfunc(const char *filename, const struct stat *stats, int flags, struct FTW *pfwt)
+{
+  (void)stats;
+  (void)flags;
+  (void)pfwt;
+  return M_remove(filename);
+}
+
+#define NFTW_LIMIT 64
+
+dboolean M_RemoveDirRecursive(const char *path)
+{
+#ifdef _WIN32
+  wchar_t *wpath;
+  int result;
+
+  SHFILEOPSTRUCTW file_op = {
+    NULL,
+    FO_DELETE,
+    NULL,
+    NULL,
+    FOF_NOCONFIRMATION | FOF_NOERRORUI | FOF_SILENT,
```

Comment:
You might want to add `FOF_NORECURSEREPARSE`.  It's deprecated on modern Windows (presumably it does not follow symlinks/junctions/shortcuts/etc. by default when deleting), but I'm not sure which version deprecated it.